src/main/java/org/commcare/formplayer/db/migration/V26__init_media_meta_data.java

```diff
@@ -0,0 +1,28 @@
+package org.commcare.formplayer.db.migration;
+
+import org.commcare.formplayer.util.Constants;
+
+import java.util.Arrays;
+
+public class V26__init_media_meta_data extends BaseFormplayerMigration {
+
+    @Override
+    public Iterable<String> getSqlStatements() {
+        return Arrays.asList(
+                "CREATE TABLE " + Constants.POSTGRES_MEDIA_META_DATA_TABLE_NAME + " (\n" +
+                        "    id varchar NOT NULL,\n" +
+                        "    filePath varchar NOT NULL,\n" +
```

Comment:
These columns names must match the names provided by the `@Column` annotation in Java. So this should be `filepath`. Some of the others also don't match.